test/extensions/filters/http/bandwidth_limit/config_test.cc

```diff
@@ -76,8 +79,11 @@ TEST(Factory, RouteSpecificFilterConfigDisabledByDefault) {
   NiceMock<Server::Configuration::MockServerFactoryContext> context;
 
   EXPECT_CALL(context.dispatcher_, createTimer_(_)).Times(0);
-  const auto route_config = factory.createRouteSpecificFilterConfig(
-      *proto_config, context, ProtobufMessage::getNullValidationVisitor());
+  const auto route_config =
```

Comment:
If you do one of these PRs again I find a straight up pointer dereference to be a bit easier to read, but not worth another pass at CI.

Reply:
noted!